docs/userGuide/contentAuthoring.md

```diff
@@ -161,6 +161,29 @@ Each variable must have an id and the value can be any MarkBind-compliant code f
 
 In any file in your site, you can include the variable by surrounding the variable's id with double curly braces, e.g. <code>{<span></span>{options}}</code>
 
+Variables can refer to other variables that are declared earlier, including helpful variables provided by MarkBind.
+
+<tip-box>
+<div>
+This is allowed, MarkBind helpful variables can be used.<br>
+
+<code>\<span id="right_hand">{<span></span>{glyphicon_hand_right}}\</span></code>
+</div>
+<div>
+This is also allowed, the second variable will be assigned the contents of the first variable.
+
+<code>\<span id="first">This is the first variable.\</span></code><br>
+<code>\<span id="second">{<span></span>{first}}\</span></code><br>
+<code>\<span id="third">{<span></span>{second}}\</span></code>
+</div>
+<div>
+This is forbidden, as the fourth variable is not declared yet.

```

Comment:
Forbidden suggests that the site would not build. Is this case?

Reply:
No, the site will continue to build, but the variable will not render properly.

Reply:
So technically the variable will be set to `undefined`? Maybe we could add on and say that your variables would not render correctly on build?